modules/navier_stokes/src/physics/WCNSFVFlowPhysics.C

```diff
@@ -478,6 +479,24 @@ WCNSFVFlowPhysics::addMomentumViscousDissipationKernels()
   }
 }
 
+void
+WCNSFVFlowPhysics::addAxisymmetricViscousSourceKernel(const std::vector<SubdomainName> & rz_blocks,
+                                                      const unsigned int radial_index)
+{
+  InputParameters params = getFactory().getValidParams("INSFVMomentumViscousSourceRZ");
+  assignBlocks(params, rz_blocks);
+  params.set<MooseFunctorName>(NS::mu) = _dynamic_viscosity_name;
```

Comment:
ok let's just add an error when there is a turbulence physics with getTurbulenceModel != none